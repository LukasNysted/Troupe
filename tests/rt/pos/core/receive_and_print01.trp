let fun f () = 
      receive [ hn x => printWithLabels x]
    val p = spawn f 
in send (p, 1) 
   
end 
