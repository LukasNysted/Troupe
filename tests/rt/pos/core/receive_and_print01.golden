2021-08-09T20:18:45.650Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: ()@{}%{}
1@{}%{}
